Source/WebCore/svg/SVGSVGElement.idl

```diff
@@ -34,9 +34,6 @@
 
     attribute float currentScale;
     [SameObject] readonly attribute SVGPoint currentTranslate;
-    
-    readonly attribute boolean useCurrentView;
-    readonly attribute SVGViewSpec currentView;
```

Comment:
We should probably also remove the C++ implementations of these properties if we're deleting them.

However, it's hard to gauge the backwards compatibility risk. How do we know it's safe to remove these properties?

Reply:
These are not part of SVG2 spec and already have been removed from other browsers. I looked into Chromium / Blink patch and patch on the bug itself and both are just touching IDL file.

Reply:
It was part of SVG 1.1, but never saw it used in the wild…

Reply:
Fixed tests and deleted, which are not needed.